[Bug] CD 과정 중 오류 발생

### 🐛 버그 설명

<!-- 발생한 버그에 대해 간단하게 설명해주세요 -->
- 작성된 workflow에서 CD 배포 진행 중 오류 발생

### 🔄 재현 단계

<!-- 버그를 재현하는 방법을 단계별로 작성해주세요 -->

1. `develop` 브랜치에 최신 변경 내용 업데이트(병합 진행)
2. `CD` workflow 진행
3. `deploy` 과정에서 문제 발생

### 📎 추가 정보 (선택)

<!-- 스크린샷, 로그, 에러 메시지 등 -->

```bash
Run appleboy/scp-action@master
  with:
    host: ***
    username: ***
    key: ***
    port: [2](https://github.com/IT-Cotato/11th-Troublog-BE/actions/runs/16234560478/job/45842938828#step:2:2)2
    source: docker-compose.yml
    target: /home/ubuntu/***/
    protocol: tcp
    timeout: [3](https://github.com/IT-Cotato/11th-Troublog-BE/actions/runs/16234560478/job/45842938828#step:2:3)0s
    command_timeout: 10m
    tar_exec: tar
    proxy_port: 22
    proxy_timeout: 30s
    curl_insecure: false
    capture_stdout: false
Run echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
  echo "$GITHUB_ACTION_PATH" >> $GITHUB_PATH
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    GITHUB_ACTION_PATH: /home/runner/work/_actions/appleboy/scp-action/master
Run entrypoint.sh
  entrypoint.sh
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    GITHUB_ACTION_PATH: /home/runner/work/_actions/appleboy/scp-action/master
    INPUT_HOST: ***
    INPUT_PORT: 22
    INPUT_PROTOCOL: tcp
    INPUT_USERNAME: ***
    INPUT_PASSWORD: 
    INPUT_PASSPHRASE: 
    INPUT_KEY: ***
    INPUT_KEY_PATH: 
    INPUT_FINGERPRINT: 
    INPUT_PROXY_HOST: 
    INPUT_PROXY_PORT: 22
    INPUT_PROXY_USERNAME: 
    INPUT_PROXY_PASSWORD: 
    INPUT_PROXY_PASSPHRASE: 
    INPUT_PROXY_KEY: 
    INPUT_PROXY_KEY_PATH: 
    INPUT_PROXY_FINGERPRINT: 
    INPUT_USE_INSECURE_CIPHER: 
    INPUT_CIPHER: 
    INPUT_PROXY_USE_INSECURE_CIPHER: 
    INPUT_PROXY_CIPHER: 
    INPUT_DEBUG: 
    INPUT_TIMEOUT: 30s
    INPUT_COMMAND_TIMEOUT: 10m
    INPUT_TARGET: /home/ubuntu/***/
    INPUT_SOURCE: docker-compose.yml
    INPUT_RM: 
    INPUT_STRIP_COMPONENTS: 
    INPUT_OVERWRITE: 
    INPUT_TAR_DEREFERENCE: 
    INPUT_TAR_TMP_PATH: 
    INPUT_TAR_EXEC: tar
    INPUT_PROXY_TIMEOUT: 30s
    INPUT_CAPTURE_STDOUT: false
    INPUT_CURL_INSECURE: false
    DRONE_SCP_VERSION: 
Downloading drone-scp-1.8.0-linux-amd6[4](https://github.com/IT-Cotato/11th-Troublog-BE/actions/runs/16234560478/job/45842938828#step:2:4) from https://github.com/appleboy/drone-scp/releases/download/v1.[8](https://github.com/IT-Cotato/11th-Troublog-BE/actions/runs/16234560478/job/45842938828#step:2:8).0
======= CLI Version Information =======
Drone SCP version 1.8.0
=======================================
drone-scp version: 1.8.0
tar all files into /tmp/QLidKsFAxu.tar.gz
tar: Cowardly refusing to create an empty archive
Try 'tar --help' or 'tar --usage' for more information.
exit status 2
Error: Process completed with exit code 2.

```



### ✅ 체크리스트

- [x] 최신 버전에서도 문제가 발생하는지 확인했습니다
- [x] 로그 파일을 확인했습니다
- [x] 비슷한 이슈가 이미 등록되어 있는지 검색했습니다
- [x] 재현 가능한 최소한의 예제를 제공했습니다

--- a/.github/workflows/cd.yml
+++ b/.github/workflows/cd.yml
@@ -30,24 +30,14 @@ jobs:
            echo "${{ secrets.ENV_FILE }}" > .env
         shell: bash
 
-      - name: Build with Gradle
-        run: |
-          chmod +x ./gradlew
-          ./gradlew bootJar
-
-      - name: Login to DockerHub
-        run: echo "${{ secrets.DOCKER_PASSWORD }}" | docker login -u "${{ secrets.DOCKER_USERNAME }}" --password-stdin
-
-      - name: Build and push Docker image
-        run: |
-          docker build -t ${{ secrets.DOCKER_REPO }}:latest .
-          docker push ${{ secrets.DOCKER_REPO }}:latest
-
   deploy:
     needs: build
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
       - name: Copy docker-compose.yml to EC2
         uses: appleboy/scp-action@master
         with:

